lib/capybara/helpers.rb

```diff
@@ -40,8 +40,8 @@ def to_regexp(text, exact: false, all_whitespace: false, options: nil)
 
     ##
     #
-    # Injects a `<base>` tag into the given HTML code, pointing to
-    # `Capybara.asset_host`.
+    # Injects a +<base>+ tag into the given HTML code, pointing to
```

Comment:
This changes breaks the docs when built locally (bundle exec rake yard) - I think rather than swapping out all of the markdown we probably just need a .yardopts file specifying that markdown is being used so rubydoc.info realizes that.

Reply:
Oh, honestly I couldn't notice the specified yard option. 😓 

https://github.com/teamcapybara/capybara/blob/87e79c2daf75035e6d5d22b43ba25cc9a7cd9920/Rakefile#L44

However sadly, `bundle exec rake yard` command breaks also `Capybara.configure` doc… ⬇️ 

![image](https://user-images.githubusercontent.com/473530/58143681-a0fe6800-7c86-11e9-93e7-a533b3566ef4.png)

Reply:
hmmm -- I wonder which breaks more things

Reply:
OK, anyway now I revert inline-code styles changed in this PR, for the future to switch to markdown styles.

In this PR, I just fix links. 💪

Reply:
About the current problem that 2 styles of YARD and Markdown are mixed, let's consider on another occasion.

Reply:
Now I rebased the commit and modified this PR's description. Please review again. 🙏